README.rst

```diff
@@ -64,42 +73,43 @@ Additional optional arguments
- encoding                      Declare the dataset encoding. If an encoding is not provided, the batch_scoring script attempts to detect it (e.g., "utf-8", "latin-1", or "iso2022_jp"). `See the Python docs for a list of valid encodings <https://docs.python.org/3/library/codecs.html#standard-encodings>`_.
- skip_dialect                  Tell the batch_scoring script to skip csv dialect detection.
-============================== ===========
+============================== ========== ========= ===========
+  Argument                     Standalone Dedicated Description
+------------------------------ ---------- --------- -----------
+ host=<host>                      \+         \+     Specifies the hostname of the prediction API endpoint (the location of the data from where to get predictions).
+ user=<user>                      \-         \+     Specifies the username used to acquire the api-token. Use quotes if the name contains spaces.
+ <import_id>                      \+         \-     Specifies imported model unique identification string. You can ask for ID your prediction administrator who did the import procedure.
+ <project_id>                     \-         \+     Specifies the project identification string. You can find the ID: embedded in the URL that displays when you are in the Leaderboard (for example, ``https://<host>/projects/<project_id>/models``) or when the prediction API is enabled, from the example shown when you click **Deploy Model** for a specific model in the Leaderboard.
+ <model_id>                       \-         \+     Specifies the model identification string. You can find the ID: embedded in the URL that displays when you are in the Leaderboard and have selected a model  (for example, ``https://<host>/projects/<project_id>/models/<model_id>``) or when the prediction API is enabled, from the example shown when you click Deploy Model for a specific model in the Leaderboard.
+ <dataset_filepath>               \+         \+     Specifies the .csv input file that the script scores. It does this by submitting prediction requests against <host> using project <project_id> and model <model_id>.
+ datarobot_key=<datarobot_key>    \-         \+     An additional datarobot_key for dedicated prediction instances. This argument is required when using on-demand workers on the Cloud platform, but not for Enterprise users.
+ password=<pwd>                   \-         \+     Specifies the password used to acquire the api-token. Use quotes if the password  contains spaces. You must specify either the password or the api_token argument. To avoid entering your password each time you run the script, use the api_token argument instead.
+ api_token=<api_token>            \-         \+     Specifies the api token for the requests; if you do not have a token, you must specify the password argument. You can retrieve your token from your profile on the **My Account** page.
```

Comment:
Specifies the API token for the requests; ...

Also, is it for multiple requests? It should either be "for requests" or "for the request"